.circleci/config.yml

```diff
@@ -304,7 +304,7 @@ commands:
         type: string
       python-version:
         type: string
-        default: "3.7.15"
+        default: "3.8.16"
```

Comment:
non-blocking: I can't think of any risks here from bumping python. ...can you? This _could_ be done in a separate commit if you're at all worried of risk.

Reply:
The only thing I know of that's holding us to 3.7 is that we're running `mypy` assuming Python 3.7 (see the `mypy.ini` files).

Reply:
I _think_ that `python_version` just tells `mypy` what set of exact rules to use when checking code (like whether `dict` is ok in type-hinting).

Reply:
Thanks!